nrich-notification/README.md

```diff
@@ -51,26 +51,26 @@ convenient way of returning a notification with a response.
 
 ## Usage
 
-nrich-notification is used nrich as a dependency of nrich-webmvc module. It is used in a `RestConstrollerAdvice`
-and on exceptions returns notifications to the client, so a common usage would be to use it in a `ControllerAdvice` to have unified error notification handling, a simple implementation of former
-advice would look something like this:
+`nrich-notification` is used nrich as a dependency of `nrich-webmvc` module. It is used in a `RestConstrollerAdvice` and on exceptions returns notifications to the client,
```

Comment:
It should be "`nrich-notification` is used as a dependency of...". There is a typo in `RestConstrollerAdvice `.

The second sentence is not clear to me, please try to simplify it.

Reply:
Please ignore the comment about the second sentence. It looks fine to me now, especially when last part is extracted to new sentence.